object-manipulation-1/src/set-value.js

```diff
@@ -0,0 +1,11 @@
+/* exported setValue */
+
+// if the trait "key" is not a part of the object,
+// then add the name of the trait into the object
+// & assign the value of that trait to the object
+
+function setValue(object, key, value) {
+  if (object.key === undefined) {
```

Comment:
I don't see anything in the instructions about only setting the property value if there was already a value, this conditional can be removed.